File: data/c#/pattern_30/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        char ch = 'A';
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            for (int j = 1; j <= num; j++)
            {
                Console.Write("{0,2:D }", ch);
                if (ch < 'Z')
                {
                    ch++;
                }
                else
                {
                    ch = (char)'A';
                }
            }
            Console.Write("\n");
        }
    }
}

File: data/c++/pattern_30/code.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int num;
    char ch = 'A';
    cout << "Enter the character of rows and columns for the square: ";
    cin >> num;

    for (int i = 1; i <= num; i++)
    {
        for (int j = 0; j < num; j++)
        {
            cout << char(ch) << " ";
            if (ch < 'Z')
            {
                ch++;
            }
            else
            {
                ch = 'A';
            }
        }
        cout << endl;
    }

    return 0;
}
